File: guanabara-java/src/com/danillo/algoritmos/lista04/Exercicio51.java
package com.danillo.algoritmos.lista04;

import java.util.Scanner;

public class Exercicio51 {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);

        System.out.println("Insira o preço de um produto: ");
        float preco = entrada.nextFloat();

        int i = 0;
        float maiorPreco = preco, menorPreco = preco;

        while(i < 3){
            System.out.println("Insira o preço de um produto: ");
            preco = entrada.nextFloat();

            if (preco > maiorPreco){
                maiorPreco = preco;
            }
            if (preco < menorPreco){
                maiorPreco = preco;
            }

            i++;
        }
        System.out.println("Maior preço inserido: R$ " +maiorPreco+ "\nMenor preço inserido: R$ " +menorPreco);
    }
}

File: guanabara-python/src/lista04/exercicio51.py
i = 0

preco = float(input("Insira o preço de um produto: "))
maior_preco = preco
menor_preco = preco

while i <= 7:
    preco = float(input("Insira o preço de um produto: "))

    if preco >= maior_preco:
        maior_preco = preco

    if preco <= menor_preco:
        menor_preco = preco

    i = i + 1

print(f"Menor preco: R$ {menor_preco}. Maior preço: {maior_preco}")
